src/query/storage/m3/storage.go

```diff
@@ -380,6 +383,33 @@ func (s *m3storage) CompleteTags(
 		wg              sync.WaitGroup
 	)
 
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Hm think the error above is already checked, maybe this was a side effect of a merge?